### ISSUE TITLE
Cannot build graphql-java-annotations with graphql-java master branch

### ISSUE THREAD

**Issue by Vincent-Pang** [Enigmatis/graphql-java-annotations#17]
When I build annotations with graphql-java master branch, the following error occur.

```
GraphQLAnnotations.java:82: error: incompatible types: invalid method reference
        }).forEach(builder::possibleType);
                   ^
    incompatible types: GraphQLType cannot be converted to GraphQLObjectType
```

I believe that it is related to this change in graphql-java
https://github.com/graphql-java/graphql-java/pull/129
The input parameter of function GraphQLUnionType -> Builder -> possibleType is changed.

```
-        public Builder possibleType(GraphQLType type) {
+        public Builder possibleType(GraphQLObjectType type) {
```

**Comment by yrashk** [comment]
Hi Vincent-Pang,

Thank you so much for highlighting this issue! 👍 

Current graphql-java-annotations' release was not checked to work against graphql-java's master. That said, I think we can definitely update this. The biggest challenge here is figuring out how often do we need to synchronize with graphql-java's master, but lets see how much burden will this be in the first place.

I am busy with other engagements right now, so if it is a high priority issue for you, would you mind submitting a PR fixing this?

**Comment by yrashk** [comment]
Closed via 6102e48d50429299dacf5fd28d58cb4504dc2ec9
